Ryujinx.Common/Utilities/UInt128Utils.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+
+namespace Ryujinx.Common.Utilities
+{
+    public static class UInt128Utils
+    {
+        public static UInt128 FromHex(string hex)
+        {
+            return new UInt128((ulong)Convert.ToInt64(hex.Substring(16), 16), (ulong)Convert.ToInt64(hex.Substring(0, 16), 16));
```

Comment:
MS decided it was a good idea to have upper as first argument, this is inverted.
```suggestion
            return new UInt128((ulong)Convert.ToInt64(hex.Substring(0, 16), 16), (ulong)Convert.ToInt64(hex.Substring(16), 16));
```